gtclang/src/gtclang/Support/ClangCompat/CompilerInvocation.h

```diff
@@ -22,7 +22,7 @@
 #include "llvm/Option/Option.h"
 
 namespace gtclang::clang_compat::CompilerInvocation {
-#if CLANG_VERSION_MAJOR < 9
+#if CLANG_VERSION_MAJOR <= 9
```

Comment:
Can you change to < 10? I think that's the pattern I usually used.

Reply:
you're right